src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsrestrictedpatientsapi/config/GsonConfig.kt

```diff
@@ -0,0 +1,55 @@
+package uk.gov.justice.digital.hmpps.hmppsrestrictedpatientsapi.config
+
+import com.google.gson.Gson
+import com.google.gson.GsonBuilder
+import com.google.gson.JsonDeserializationContext
+import com.google.gson.JsonDeserializer
+import com.google.gson.JsonElement
+import com.google.gson.JsonPrimitive
+import com.google.gson.JsonSerializationContext
+import com.google.gson.JsonSerializer
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import java.lang.reflect.Type
+import java.time.LocalDate
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
+import java.util.Locale
+
+@Configuration
+class GsonConfig() {
+  @Bean
+  fun gson(): Gson {
```

Comment:
I'm guessing you grabbed this config from some other project.  Seems a little bit naff, normally I think you can just add a `GsonBuilderCustomizer` to customise and retain the spring boot goodness rather than replacing the whole bean. see https://github.com/spring-projects/spring-boot/blob/main/spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/gson/GsonAutoConfiguration.java